cmd/system-probe/windows_resources/system-probe-msg.mc

```diff
@@ -104,3 +104,11 @@ Severity=Error
 Language=English
 The service failed to start. Error %1
 .
+
+MessageId=15
+SymbolicName=MSG_SYSPROBE_RESTART_INACTIVITY
+Severity=Warning
+Language=English
+System probe restarting afer %1.  The process agent has not queried for data.  It may not be configured correctly and/or running.
```

Comment:
typo: after

Reply:
This duration is a period of inactivity as well, not necessarily how long since system-probe start.